package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ripple-binary-codec",
-  "version": "0.1.5",
+  "version": "0.1.6",
```

Comment:
Better to bump the version after the PR is merged with a separate commit (that also includes the git tags)